gatsby/createPages.ts

```diff
@@ -27,7 +27,7 @@ export default async ({ graphql, actions }: any) => {
     });
 
     if (node.addresses !== null && node.addresses.length) {
-      node.addresses.forEach(addr => {
+      node.addresses.forEach((addr, key) => {
```

Comment:
is this actually being used here at all?

Reply:
The `key` param? No, but maybe later - though it doesn't really affect functionality right now. Can remove if you prefer

Reply:
Just worried about unused stuff resulting in confusion for anyone who comes back to look at this later. It's not really a significant issue, just something to consider.